### DISCUSSION TITLE
2024/12/27/mastodon%E7%AC%AC%E4%BA%8C%E9%83%A8%E5%88%86/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#8] [Announcements]
# 2024/12/27/mastodon%E7%AC%AC%E4%BA%8C%E9%83%A8%E5%88%86/

开启 Mastodon 的搜索功能，以及结合 Cloudflare CDN 与 Tunnel 保护源服务器。

https://blog.byteloid.one/2024/12/27/mastodon%E7%AC%AC%E4%BA%8C%E9%83%A8%E5%88%86/

**Comment by liuzhen9320** [comment]
很棒欸！通过 Cloudflare Tunnel 代理流量，完美解决服务器暴露在公网上被扫描的难题
